.github/workflows/android_main.yml

```diff
@@ -118,8 +119,6 @@ jobs:
 
     - name: conan remote
       run: conan remote add --index 0 odr https://artifactory.opendocument.app/artifactory/api/conan/conan
-    - name: conan login
```

Comment:
Why was that deleted? On line 175 we do upload some binaries...

Reply:
Somehow I've missed that part. I was assuming we only upload binaries in build, not test jobs. fixed now.